cluster/examples/vmi-slirp.yaml

```diff
@@ -19,12 +19,11 @@ spec:
         volumeName: cloudinitvolume
       interfaces:
       - name: testSlirp
-        slirp:
-          ports:
-          - name: http
-            podPort: 80
-            port: 80
-            protocol: TCP
+        ports:
+        - name: http
+          port: 80
+          protocol: TCP
+        slirp: {}
```

Comment:
I more and more have the feeling that `slirp` and `bridge` should go to the `network` section under `pod`. Maybe we can discuss that too outside of this PR. @booxter @SchSeba.

Reply:
I think it will be best to speak about it outside of this PR. We may also create a post about it in the kubevirt group to get more comments.